pkg/operator2/starter.go

```diff
@@ -132,6 +137,20 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		resourceSyncer,
 	)
 
+	clusterOperatorStatus := status.NewClusterOperatorStatusController(
+		"openshift-authentication",
+		[]configv1.ObjectReference{
```

Comment:
This is incomplete, needs to have:

The 3 top level configs we look at
The 4 namespaces we look at
The operator config